test/models/confirmable_test.rb

```diff
@@ -77,6 +77,24 @@ def setup
     assert_equal "can't be blank", confirmed_user.errors[:confirmation_token].join
   end
 
+  test 'should return a new record with errors when a blank token is given and a record exists on the database' do
```

Comment:
I think this no longer needs 'and a record exists on the database'

Reply:
I think it's better to keep since that's the behavior we're trying to ensure it doesn't happen anymore.
I know that since the current code validates the parameter before doing anything else, it's unlikely for this to happen, but this test ensures we don't add regressions in the future.